01.general/02.how-to-create-your-own-ad-filters/docs.en.md

```diff
@@ -427,6 +427,24 @@ There is a set of modifiers, which can be used to limit the rule's application a
 * `||example.org^$script,stylesheet` — corresponds to all the scripts and styles from `example.org`.
 * `||example.org^$~image,~script,~stylesheet` — corresponds to all requests to `example.org` except for the images, scripts and styles.
 
+<a id="document-modifier"></a>
+##### **`document`**
+
+The rule corresponds to HTML document requests, it applies to main frames only (i.e. HTML documents).
+
+In the case of blocking rules, `$document` also disables "main frame bypass" logic and blocks requests _strictly_ with a blocking page.
```

Comment:
```suggestion
By default, AdGuard won't block the requests that are loaded in the browser tab (e.g. "main frame bypass"). The idea is not to prevent pages from loading as the user clearly indicated that they want this page to be loaded. However, if the `$document` modifier is specified explicitly, AdGuard does not use that logic and prevents the page load. Instead, it responds with a "blocking page".
```